src/main/groovy/enum/PersonType.groovy

```diff
@@ -0,0 +1,21 @@
+package com.mini.asaas
+
+import grails.util.Holders
+
+enum PersonType {
+    NATURAL,
+    LEGAL
+
+    public static PersonType convert(String personType) {
+        try {
+            if (personType instanceof String) personType = personType.toUpperCase()
+            return personType as PersonType
+        } catch (Exception e) {
+            return null
+        }
+    }
+
+    public String getLabel() {
+        return Holders.applicationContext.getBean("messageSource").getMessage("PersonType.${this}.label", null, "", new Locale("pt", "BR"))
```

Comment:
Muito bom! É boa prática mapear essas mensagens no arquivo de `messages`.
Entretanto, é um tanto verboso todo esse código necessário pra trazer a mensagem, e isso provavelmente vai ser repetido várias vezes durante o projeto. 

O que acham de fazer uma classe utilitária pra simplificar isso?

Algo que vocês pudessem usar como `MessageSourceUtils.getMessage("PersonType.${this}.label")`
(apenas exemplo)